core/templates/domain/entity_editor_browser_tabs_info/entity-editor-browser-tabs-info-domain.constants.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Constants for entity editor browser tabs info domain.
+ */
+
+export const EntityEditorBrowserTabsInfoDomainConstants = {
+  OPENED_EXPLORATION_EDITOR_PAGES: 'opened_exploration_editor_pages',
```

Comment:
Either call it a "page" or a "browser tab" throughout, not both. Probably the latter.

Reply:
Done